### ISSUE TITLE
Anomalies in the passiv data

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/ocf_datapipes#141]
Plot of max valye per day
<img width="282" alt="Screenshot 2023-01-12 at 08 05 29" src="https://user-images.githubusercontent.com/34686298/212011652-22719a3a-fc28-4ec8-a057-2132e8f4f911.png">

suggestions:
- capacity could defined 99% of data, remove anaything above
- All could remove 5stds away from the mean

Note any removal of data will cause gaps in the data

**Comment by jacobbieker** [comment]
What about taking the mean value of the points around the anomaly and replacing it with that? Would at least keep the gaps filled

**Comment by peterdudfield** [comment]
> What about taking the mean value of the points around the anomaly and replacing it with that? Would at least keep the gaps filled

yea, or just linear interpolate. Depends how big the anomly is. If its > 12 hours, then probably just remove the data. If its one point, i agree, can just be filled in. But data pipes would be able to handle the gap in data. 

Whenever I get a chance, Ill try to point out the advantages to ocf datapipes

**Comment by peterdudfield** [comment]
@simlmx reference here

**Comment by jacobbieker** [comment]
Yeah, currently for training if there is a gap, it'll mean the valid time period selection will say there is an issue, but yeah, I agree if its for awhile, then just remove it all.

**Comment by simlmx** [comment]
Making sure to use absolute loss `|y - pred|` instead of squared loss `(y-pred)**2` when training my model fixed all the issues with outliers. When using absolute loss, removing the problematic values didn't help in a significant way.

**Comment by dfulu** [comment]
This might be partially solved now. Using a percentile for the estimated capacity was added in #225. the mentioned filtering isn't included yet
